wpcsite/settings.py

```diff
@@ -242,19 +266,31 @@ def rel_path(*subpaths):
         ),
         'APP_DIRS': True,
         'OPTIONS': {
-            'context_processors': [
-                'django.contrib.auth.context_processors.auth',
-                'django.template.context_processors.debug',
-                'django.template.context_processors.i18n',
-                'django.template.context_processors.media',
-                'django.template.context_processors.static',
-                'django.template.context_processors.tz',
-                'django.contrib.messages.context_processors.messages',
-            ]
+            'context_processors': _CONTEXT_PROCESSORS,
         }
     },
 ]
 
+# L10n extraction configuration
+PUENTE = {
+    'BASE_DIR': BASE_DIR,
+    'DOMAIN_METHODS': {
+        'django': [
+            ('**/migrations/**', 'ignore'),
+
+            ('bcauth/**.py', 'python'),
+            ('bcauth/**/jinja2/**.html', 'jinja2'),
+
+            ('mdn/**.py', 'python'),
+            ('mdn/**/jinja2/**.html', 'jinja2'),
+
+            ('webplatformcompat/**.py', 'python'),
+            ('webplatformcompat/**/jinja2/**.html', 'jinja2'),
+        ]
+    },
+    'PROJECT': 'Mozilla Browsercompat',
```

Comment:
This title gets put at the top of the resulting `.pot` file. It should probably match whatever it's called in Pontoon.